Fonts/PTSenif-Bold/specimen_files/grid_12-825-55-15.css

```diff
@@ -0,0 +1,450 @@
+/* Notes about grid:
+Columns:      12
+Grid Width:   825px
+Column Width: 55px
+Gutter Width: 15px
+------------------------------- */
+
+.section { margin-bottom: 18px;
+}
+
+.section:after { content: ".";display: block;height: 0;clear: both;visibility: hidden;
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)